src/main/java/com/iota/iri/service/milestone/impl/LatestSolidMilestoneTrackerImpl.java

```diff
@@ -103,23 +115,24 @@
      * </p>
      * <p>
      * Note: Instead of handing over the dependencies in the constructor, we register them lazy. This allows us to have
-     *       circular dependencies because the instantiation is separated from the dependency injection. To reduce the
-     *       amount of code that is necessary to correctly instantiate this class, we return the instance itself which
-     *       allows us to still instantiate, initialize and assign in one line - see Example:
+     * circular dependencies because the instantiation is separated from the dependency injection. To reduce the amount
+     * of code that is necessary to correctly instantiate this class, we return the instance itself which allows us to
+     * still instantiate, initialize and assign in one line - see Example:
      * </p>
-     *       {@code latestSolidMilestoneTracker = new LatestSolidMilestoneTrackerImpl().init(...);}
+     * {@code latestSolidMilestoneTracker = new LatestSolidMilestoneTrackerImpl().init(...);}
      *
-     * @param tangle Tangle object which acts as a database interface
-     * @param snapshotProvider manager for the snapshots that allows us to retrieve the relevant snapshots of this node
-     * @param milestoneService contains the important business logic when dealing with milestones
-     * @param ledgerService the manager for
+     * @param tangle                 Tangle object which acts as a database interface
+     * @param snapshotProvider       manager for the snapshots that allows us to retrieve the relevant snapshots of this
+     *                               node
+     * @param milestoneService       contains the important business logic when dealing with milestones
+     * @param ledgerService          the manager for
      * @param latestMilestoneTracker the manager that keeps track of the latest milestone
-     * @param transactionRequester the manager which keeps and tracks transactions which are requested
+     * @param transactionRequester   the manager which keeps and tracks transactions which are requested
      * @return the initialized instance itself to allow chaining
      */
     public LatestSolidMilestoneTrackerImpl init(Tangle tangle, SnapshotProvider snapshotProvider,
-            MilestoneService milestoneService, LedgerService ledgerService,
-            LatestMilestoneTracker latestMilestoneTracker, TransactionRequester transactionRequester) {
+                                                MilestoneService milestoneService, LedgerService ledgerService,
+                                                LatestMilestoneTracker latestMilestoneTracker, TransactionRequester transactionRequester) {
```

Comment:
This goes over the 120 columns limit
When I do auto-formatting on my IDE I get a different result :man_shrugging: 

Are you using the auto-formatting instructions from https://github.com/iotaledger/iri/blob/dev/STYLEGUIDE.md#formatting?

Reply:
Yes but anyways, I reverted all the accidental reformats.